.github/workflows/delete-gcp-resources.yml

```
@@ -73,13 +73,15 @@ jobs:
 
       # Deletes cache images older than $DELETE_AGE_DAYS days.
       #
-      # Keeps the latest $KEEP_LATEST_IMAGE_COUNT images of each type:
+      # Keeps an extra $KEEP_LATEST_IMAGE_COUNT images of each type:
```

```suggestion
      # Keeps an extra $KEEP_LATEST_IMAGE_COUNT of the most recent images that are older than $DELETE_AGE_DAYS of each type:
```

---

That's helpful, but maybe we could spell out exactly what happens:
```suggestion
      # Keeps all images younger than $DELETE_AGE_DAYS.
      # Also keeps $KEEP_LATEST_IMAGE_COUNT older images of each type:
```